include/blas_quda.h

```diff
@@ -293,7 +294,7 @@ namespace quda {
 
     inline array<double, 2> max_deviation(const ColorSpinorField &x, const ColorSpinorField &y)
```

Comment:
Design idea, thoughts on adding a third optional argument `int n = 0` or something that will return element `[n]` instead of the hard-coded `[0]`? Whether or not it's ever needed is another story... but may as well have that support in there for convenience?

Reply:
Oh, nevermind, these are the versions taking in a _single_ `ColorSpinorField`. Please disregard.